scripts/ollama-docker.sh

```
@@ -0,0 +1,461 @@
+    print_status "Testing Ollama installation..."
+    
+    # Test basic API
+    if curl -s http://localhost:11434/api/tags > /dev/null 2>&1; then
+        print_success "Ollama API is responding"
+    else
+        print_error "Ollama API is not responding"
+        return 1
+    fi
+    
+    # Test model list
+    local model_count=$(curl -s http://localhost:11434/api/tags | jq '.models | length' 2>/dev/null || echo "0")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)